deluge/ui/ui_entry.py

```diff
@@ -32,65 +34,92 @@ def start_ui():
     """Entry point for ui script"""
     lang.setup_translations()
 
-    # Setup the argument parser
-    parser = BaseArgParser()
-    group = parser.add_argument_group(_("UI Options"))
-
+    # Get the registered UI entry points
     ui_entrypoints = dict([(entrypoint.name, entrypoint.load())
                            for entrypoint in pkg_resources.iter_entry_points("deluge.ui")])
+    ui_titles = sorted(ui_entrypoints.keys())
 
-    cmd_help = [_("The UI that you wish to launch. The UI choices are:")]
-    cmd_help.extend(["%s -- %s" % (k, getattr(v, "cmd_description", "")) for k, v in ui_entrypoints.iteritems()])
-
-    group.add_argument("-u", "--ui", action="store",
-                       choices=ui_entrypoints.keys(), help="\n  * ".join(cmd_help))
-    group.add_argument("-a", "--args", action="store",
-                       help=_('Arguments to pass to the UI. Multiple args must be quoted, e.g. -a "--option args"'))
-    group.add_argument("-s", "--set-default-ui", dest="default_ui", choices=ui_entrypoints.keys(),
-                       help=_("Sets the default UI to be run when no UI is specified"), action="store")
+    # Setup the initial argument parser for the UI selection
```

Comment:
This comment should perhaps mention that the arg results are processed here but the text will not be shown and reusing the group later in code is required for displaying to the user.